src/Compute/Compute/help/New-AzVM.md

```diff
@@ -1154,7 +1154,8 @@ Accept wildcard characters: False
 ```
 
 ### -Zone
-Specifies the zone list of the virtual machine.
+Specifies the zone of the virtual machine. Although it takes in an array of zones, virtual machines do not support multiple availability zones.
```

Comment:
>zone [](http://example.com/codeflow?start=14&length=4)

`Azure availability zone`: it seems more accurate.

Extra points: update the cmdlet to make `-AvailabilityZone` an alias for `-Zone` (it would be a 1-line change) :-)

Reply:
we will not be adding the alias at the moment. We will consider adding that for all cmdlets that use Zone in the future.